src/server/pfs/fuse/filesystem.go

```diff
@@ -288,12 +288,22 @@ func (f *file) Fsync(ctx context.Context, req *fuse.FsyncRequest) error {
 	return nil
 }
 
+func (f *file) delimiter() pfsclient.Delimiter {
+	if strings.HasSuffix(f.File.Path, ".json") {
+		return pfsclient.Delimiter_JSON
+	}
+	if strings.HasSuffix(f.File.Path, ".blob") {
```

Comment:
Minor but I think `.bin` is a little more common than `.blob` for raw binary data. Could we do that instead.